src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,603 @@
+﻿// <copyright file="AuthorizationServersScenarios.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    [Collection(nameof(AuthorizationServersScenarios))]
+    public class AuthorizationServersScenarios
+    {
+        private const string SdkPrefix = "dotnet_sdk";
+
+        [Fact(Skip = "Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
+        public async Task CreateAuthorizationServer()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:{nameof(DeleteAuthorizationServer)}:Test Authorization Server ({TestClient.RandomString(6)})";
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            createdAuthorizationServer.Should().NotBeNull();
+            createdAuthorizationServer.Name.Should().Be(testAuthorizationServerName);
+            createdAuthorizationServer.Description.Should().Be("Test Authorization Server");
+            createdAuthorizationServer.Audiences.Count.Should().Be(1);
+            createdAuthorizationServer.Audiences[0].Should().Be("api://default");
```

Comment:
We should delete the createdAuthorizationServer once we finish with the asserts.

Reply:
Good catch!  I'll update.